src/test/scala/com/exasol/cloudetl/kinesis/KinesisUserPropertiesTest.scala

```diff
@@ -0,0 +1,136 @@
+package com.exasol.cloudetl.kinesis
+
+import com.exasol.cloudetl.kinesis.KinesisUserProperties._
+
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.funsuite.AnyFunSuite
+
+class KinesisUserPropertiesTest extends AnyFunSuite with BeforeAndAfterAll {
+  private[this] val kinesisProperties = Map(
+    "TABLE_NAME" -> "TEST_TABLE",
+    "AWS_ACCESS_KEY" -> "MY_ACCESS_KEY",
+    "AWS_SECRET_KEY" -> "MY_SECRET_KEY",
+    "AWS_SESSION_TOKEN" -> "MY_SESSION_TOKEN",
+    "STREAM_NAME" -> "Test_stream",
+    "REGION" -> "eu-west-1"
+  )
+
+  override final def beforeAll(): Unit =
```

Comment:
remove `beforeAll` and `BeforeAndAfterAll`.

Reply:
Nice findings. Forgot about it